src/lib/flags.ts

```diff
@@ -1,6 +1,17 @@
 import { Args, Flags } from '@oclif/core';
 
+export const branchFlag = (additionalDescription: string[] = []) => ({
+  branch: Flags.string({
+    aliases: ['version'],
+    default: 'stable',
+    deprecateAliases: true,
+    description: ['Defaults to `stable` (i.e., your main project version).'].concat(additionalDescription).join(' '),
+    summary: 'ReadMe project version',
+  }),
+});
```

Comment:
might be jumping ahead here, but eventually do we plan on updating `rdme` to support users uploading oas specs to a git branch as opposed to a version?

is that work ticketed somewhere or should we start updating the description for the branch flag here in this PR with examples like `--branch=1.0.0_some-branch`?